gpversionchecker/src/main/java/com/robohorse/gpversionchecker/utils/DataParser.java

```diff
@@ -18,9 +19,18 @@
     private static final String DIV_DESCRIPTION = "div[itemprop=description]";
 
     public Version parse(Document document, final String currentVersion, final String url) {
-        final String newVersion = document.select(DIV_VERSION)
-                .first()
-                .ownText();
+         String newVersion ="";
+        if (document != null) {
+            Elements element = document.getElementsContainingOwnText("Current Version");
```

Comment:
This code will work on the app html page ENGLISH version only( But, that is good idea - we can make 2 requests to make this library alive again!

Reply:
i will do some changes later and publish new version! Thank you!

Reply:
Please update it as soon as possible.

Reply:
fixed in 1.6.0

Reply:
Thanks,